### DISCUSSION TITLE
Redownloading a map after local edits

### DISCUSSION THREAD

**Discussion by Gyoshi** [ppy/osu#23703] [Q&A]
After editing a map I want to be able to easily redownload the 'official' version. Basically this https://github.com/ppy/osu/pull/19230 but for maps with local edits.

It's also unfortunate that your replays seem to disappear after making edits (and redownloading). If they're not practically recoverable, I'd appreciate it if there were at least a bigger warning about your replays being deleted if you save any changes.

**Comment by bdach** [comment]
> Basically this https://github.com/ppy/osu/pull/19230 but for maps with local edits

I don't understand what this means.

This flow is currently a bit broken (https://github.com/ppy/osu/issues/23524) but that will be revisited after some robustness checks are in place to prevent players from submitting plays from tampered beatmaps.

**Reply by Gyoshi** [reply]
Ok that is really strange, I just now was able to update a map with local edits when a change was uploaded by the mapper. But that is not what I am talking about. 

What I want is effectively to be able to discard local edits. Example: 
Make experimental change -> give modding feedback -> redownload map without local edits

**Reply by bdach** [reply]
Then that is covered by the issue above.
